README.md

```diff
@@ -2,43 +2,87 @@
 
 🏗 A CLI to setup and manage Matic validator nodes
 
-### Installation
+### Installation (Host Machine)
 
 Please make sure you have installed following dependencies:
 
-* Git
-* Node/npm v10.17.0 (or higher)
+* Build Essentials
+    ```bash
+    sudo apt install build-essential
+    ```
+* Node v10.17.0
+    ```bash
+    curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+    nvm install 10.17.0
+    ```
 * Go 1.18+
-* Docker (only if running on a docker based setup)
-* Rabbitmq (Latest stable version, https://www.rabbitmq.com/download.html) (Not required for docker based setup)
-* Solc v0.5.11 (https://solidity.readthedocs.io/en/v0.5.3/installing-solidity.html#binary-packages) (Not required for docker based setup)
-* Ganache CLI (https://www.npmjs.com/package/ganache-cli) (Not required for docker based setup)
+    ```bash
+    wget https://raw.githubusercontent.com/maticnetwork/node-ansible/master/go-install.sh
+    bash go-install.sh --remove
+    bash go-install.sh
+    ```
+* Docker (https://docs.docker.com/engine/install/linux-postinstall/)
+* Solc v0.5.16
+    ```bash
+    sudo snap install solc
+    ```
+* Python 2
+    ```bash
+    sudo apt install python2
+    alias python="/usr/bin/python2"
```

Comment:
python2 is deprecated: https://www.python.org/doc/sunset-python-2/ 
Is there any reason not using python3 here?